api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -180,6 +180,10 @@ message ExternalProcessor {
   // :ref:`clear_route_cache <envoy_v3_api_field_service.ext_proc.v3.CommonResponse.clear_route_cache>`
   // field is set in an external processor response.
   bool disable_clear_route_cache = 11;
+
+  // [#not-implemented-hide:]
+  // Allow headers matching the ``forward_rules`` to be forwarded to the ext_proc server.
```

Comment:
```suggestion
  // Allow headers that match the ``forward_rules`` to be forwarded to the external processing server.
```
nit, To be consistent with name `external processing server`/ or `external processor` throughout this filter

Reply:
done